services/softlayer_network_storage.go

```diff
@@ -38,23 +38,40 @@ func (slns *softLayer_Network_Storage_Service) CreateIscsiVolume(size int, locat
 		return datatypes.SoftLayer_Network_Storage{}, errors.New("Cannot create negative sized volumes")
 	}
 
-	iscsiVolumeItemId, err := slns.getIscsiVolumeItemIdBasedOnSize(size)
+	sizeItemPriceId, err := slns.getIscsiVolumeItemIdBasedOnSize(size)
 	if err != nil {
 		return datatypes.SoftLayer_Network_Storage{}, err
 	}
 
+	iopsItemPriceId := slns.getPerformanceStorageItemPriceIdByIops(size)
+
 	order := datatypes.SoftLayer_Product_Order{
 		Location:    location,
-		ComplexType: "SoftLayer_Container_Product_Order",
+		ComplexType: "SoftLayer_Container_Product_Order_Network_PerformanceStorage_Iscsi",
+		OsFormatType: datatypes.OsFormatType{
+			Id:      12,
```

Comment:
`OsFormatType` is a mandatory argument to place a consistent performance disk order in Softlayer, and it has fixed values `Id : 12 , Keyname: Linux` for performance storage.